### ISSUE TITLE
Review user-centered report evaluation results

### ISSUE THREAD

**Issue by jackiekazil** [econ-ark/HARK#135]
Hey all -- Like I said, I was working on reviewing all the things in econ-ark / HARK to try to focus on improving the project to both users and contributing users. This doc is not done, but I think it is good jumping off point. 

https://docs.google.com/document/d/1MztmMO0IJy-xagnluJjqZmPrVkI1pAAfp3Lm-g6ijNY/edit# 
(Please explicitly ask for permission. I didn't just want to open to the world.) 

2 things to do in this ticket to consider it "Done": 

1. Review the doc and add to it if anything is missing. 
2. Go through the doc and the existing tickets and add links to the doc to the tickets if something is already in progress.

**Comment by jackiekazil** [comment]
P.S. I used a checklist from a talk I have done a couple of times to create this list. I was thinking of taking this evaluation and turning it into a blog post and using HARK as an example.... it has a solid start, but just hasn't done this before. Is there a specific blog that you think might go well on? (if not, I can post it on medium.)

**Comment by mnwhite** [comment]
Quick thing: Where it's relevant, I refer to branches by name in Issues, where applicable.  But I didn't know you could associate a branch with an Issue.  How do you do that?

**Comment by jackiekazil** [comment]
@mnwhite I didn't realize that was happening. I was looking at it the other way.... 
here is a screenshot of branches... 
![Uploading Screen Shot 2017-12-06 at 9.21.47 PM.png…]()

I was approaching it from... how do I know which branch goes with which ticket? 
Let's take your approach for OpenCL-dev branch.
For the sake of this description, let's say this is the ticket https://github.com/econ-ark/HARK/issues/136 - (Assumes that is the right ticket and someone forgot to explicitly add it.) 
^^ If that is the case, then that is valid approach. When I wrote that, I was thinking... OpenCL-dev.136 and putting the ticket number in the branch name.

**Comment by mnwhite** [comment]
First: Now that I look around, I can only find a couple examples of what I referred to above.  There are probably older ones on closed issues.

But now I understand what you're talking about.  Going forward, that is a good policy.  I've tried to make issues to describe what I'm working on in the dev branches, but there's no indication of that from the branches themselves.

As your document alludes to, this will be more visible / transparent when we have a real webpage with a What's Happening (or whatever the kids are saying these days) section with descriptions of work in progress.
